### PR TITLE
feature: Extend sealer save to support multi-image-archive and tmp-dir

### LINKED ISSUE #2147: Update containers vendor
## Issue Description

1.  support set load tmp dir for save and load, we could set `tmp-dir` arg to `sysContext.BigFilesTemporaryDir`.
2. Waiting for a separate functions from containers SDK, which will not guess what type of input tar file. this is more time-saving.

more to see : https://github.com/containers/common/issues/1376

Type: *feature request*

### PR DESCRIPTION
**Pull request by cubxxw**
### Does this pull request fix one issue?

Fixes #2147

### CODE REVIEW

File: cmd/sealer/cmd/image/load.go [hunk 1]
```diff
@@ -58,7 +62,7 @@ func NewLoadCmd() *cobra.Command {
 	flags := loadCmd.Flags()
 	flags.StringVarP(&loadOpts.Input, "input", "i", "", "Load image from file")
 	flags.BoolVarP(&loadOpts.Quiet, "quiet", "q", false, "Suppress the output")
-	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "set temporary directory when load image. if not set, use system`s temporary directory")
+	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "d", "Set temporary directory when load image. if not set, use system temporary directory(`/var/tmp/`)")
```
Comment (Stevent-fei) [inline cmd/sealer/cmd/image/load.go:65]: `-d` will conflict with debug mode?
`set` should be lowercase

Comment (cubxxw) [reply cmd/sealer/cmd/image/load.go:65]: I'll go back and finish it off. `-d, --debug`

File: cmd/sealer/cmd/image/save.go [hunk 2]
```diff
@@ -42,31 +49,46 @@ func NewSaveCmd() *cobra.Command {
...
+	// TODO: Waiting for implementation, not yet supported
+	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "Set temporary directory when load image. if not set, use system temporary directory(`/var/tmp/`)")
+
 	flags.BoolVarP(&saveOpts.Quiet, "quiet", "q", false, "Suppress the output")
-	flags.BoolVar(&saveOpts.Compress, "compress", false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
+
+	compressFlagName := "compress"
+	flags.BoolVar(&saveOpts.Compress, compressFlagName, false, "Compress tarball image layers when saving to a directory using the 'dir' transport. (default is same compression type as source)")
+
+	MultiImageArchiveFlagName := "multi-image-archive"
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/save.go:71]: its good to try adding more flags to make sealer more powerful , while i have some different advise : 

on pr do one thing, could we split it as two PR, one is adding  `tmp-dir` flag , one is adding another ?

File: cmd/sealer/cmd/image/load.go (hunk 1 above)
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/load.go:65]: use system temporary directory is not (`/var/tmp/`)at present .

**Comment by cubxxw** [comment]
I'm not really confused here why did it go wrong
https://github.com/sealerio/sealer/actions/runs/4592489506/jobs/8109571732?pr=2160#step:7:18

**Comment by Stevent-fei** [comment]
> I'm not really confused here why did it go wrong https://github.com/sealerio/sealer/actions/runs/4592489506/jobs/8109571732?pr=2160#step:7:18

image

**Pull request closed without merging**